src/pages/AddNewCake.css

```diff
@@ -0,0 +1,19 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 1rem;
+  background: var(--color-background);
+}
+
+.btn-submit {
+  background-color: var(--color-text);
+  border: 0px solid white;
```

Comment:
if it's a 0px border and you don't plan to transition it in some way, you could just write border: none;

Reply:
implemented ! thx!